frontend/src/services/clientServerApi.js

```
@@ -0,0 +1,172 @@
+      // fetchPost now returns an object like { success: boolean, data: ..., errors: ..., message: ... }
+      const response = await fetchPost(
+         `${BACKEND_URL_AUTH}/register`,
+         credentials
+      );
+
+      if (!response.success) {
+         return response;
+      }
+
+      // --- Added: Update authStore on successful registration ---
+      if (response.data && response.data.userId) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)